packages/ui-svelte/tailwind.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    // `content` is replaced instead of extended, so this line has to be added in
+    // the `content` of each app' tailwind.config.js
+    content: ['node_modules/@vercel/examples-ui/ui/**/*.{js,ts,jsx,tsx}'],
```

Comment:
This feels rather brittle, and a poor user experience. Does the component library really need to use Tailwind?